src/App.css

```diff
@@ -49,28 +78,89 @@
 
 .nav-menu a {
   text-decoration: none;
-  color: #000;
+  width: 3.688rem;
+  height: 1rem;
+  margin: 1.125rem 2.563rem 0.688rem 0;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: var(--nav-menu-theme);
 }
 
 .nav-menu a:hover {
-  text-decoration: underline;
+  text-decoration: none;
+}
+
+.person-icon {
+  align-items: center;
```

Comment:
- [ ] The same applies to the `align-items: center;` style. It is repeated 14 times. It would be more efficient to create a class and then reuse that className where it applies to the elements. Check the comment above for an example